### DISCUSSION TITLE
How to reset the tweaks applied by CTT Tool.

### DISCUSSION THREAD

**Discussion by Lakshay500** [ChrisTitusTech/winutil#1692] [Q&A]
Hello! I used the Chris Titus Tech Tool (CTT) about 2 days ago. I loved the software, but it isn't for me (as I'm a newbie). Is there a way to reset the tweaks and turn my laptop back to how it was before using the tool? I'm not sure if this was caused by using the tool but I have been experiencing system slowing down at the login screen & game crashes. I don't have a clue whether or not if it's a hardware issue or it's something related to CTT. Could anyone please, help?

**Comment by hirschsystems** [comment]
You can hit the "Get Installed" button at the top of the tweaks page, and give it a second to populate the applied tweaks. Deselect any you wish to keep, and hit the "Undo Selected Tweaks" button at the bottom.
![image](https://github.com/ChrisTitusTech/winutil/assets/163751710/a96b541a-b684-4c71-995f-5af797433960)
![image](https://github.com/ChrisTitusTech/winutil/assets/163751710/7747d6ee-27be-4e14-a545-d48952138a66)

**Reply by sslimesito** [reply]
I need helppp I ran some tweaks and now my program that injects into my game to give me fov slider auto closes everytime what do i do

**Reply by Revengerz1** [reply]
> You can hit the "Get Installed" button at the top of the tweaks page, and give it a second to populate the applied tweaks. Deselect any you wish to keep, and hit the "Undo Selected Tweaks" button at the bottom. ![image](https://private-user-images.githubusercontent.com/163751710/313711106-a96b541a-b684-4c71-995f-5af797433960.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTI5MzMxNjMsIm5iZiI6MTc1MjkzMjg2MywicGF0aCI6Ii8xNjM3NTE3MTAvMzEzNzExMTA2LWE5NmI1NDFhLWI2ODQtNGM3MS05OTVmLTVhZjc5NzQzMzk2MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcxOVQxMzQ3NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZGYzZWZlNGU5M2IwMjc3NzY2ZDUzZTM4Y2IyODViNjc1NWNhY2I4ZDMyOTI0YzE1NWJjODRhZWM3Y2ZkMTE4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Kq6qUGrwLgLc9SGJdHLw6T8Rkl-OborA3Kn7NQSy3qg) ![image](https://private-user-images.githubusercontent.com/163751710/313711555-7747d6ee-27be-4e14-a545-d48952138a66.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTI5MzMxNjMsIm5iZiI6MTc1MjkzMjg2MywicGF0aCI6Ii8xNjM3NTE3MTAvMzEzNzExNTU1LTc3NDdkNmVlLTI3YmUtNGUxNC1hNTQ1LWQ0ODk1MjEzOGE2Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcxOVQxMzQ3NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNWRhOTExZWQ5MjhiZGZkYzRhYzNhOTVjYjljNTA0ZTgzYWIzZmJhYmZjMDIwODEyZmY2N2FjYjYzYTYyNmMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ACACq5kX_UiiuS4smFyrvPIkRQA5MnVN5rj44KQm6OM)


that resets the toggle tweeks aswell? it fucked up my sleep mode. my computer cant go to sleep anymore

**Comment by sslimesito** [comment]
I need helppp I ran some tweaks and now my program that injects into my game to give me fov slider auto closes everytime what do i do

**Reply by ISB333** [reply]
I'm clearly coming too late, but anyway maybe it will help others one.
I think the tools create a restore point, I didn't do it myself and since I've using lot of tools to optimize w11, I'm not 100% sure which one created a restore point but I think it's the Chris Titus tool.

For those who haven't started to use the tool, create a restore point, maybe the tool works for some people idk. But for me it was awful and fucked up everything, CS2, Brave, Powershell, File manager, everything was so fucking slow it was like if my computer have loose more than half of RAM power. Since I came back to restore point, everything works as fine as before.

**Comment by rdmguy23** [comment]
i need help, i ran a tweak now windows just refuses to let me move the taskbar icons to the left.
